tee-worker/litentry/core/data-providers/src/discord_litentry.rs

```diff
@@ -130,23 +130,14 @@ impl DiscordLitentryClient {
 	// user has role in channel
 	pub fn has_role(
 		&mut self,
-		guild_id: Vec<u8>,
 		role_id_s: String,
 		handler: Vec<u8>,
 	) -> Result<DiscordResponse, Error> {
-		let guild_id_s = vec_to_string(guild_id)?;
 		let handler_s = vec_to_string(handler)?;
-		debug!(
-			"discord check id_hubber, guild_id: {}, role_id: {}, handler: {}",
-			guild_id_s, role_id_s, handler_s
-		);
+		debug!("discord check id_hubber, role_id: {}, handler: {}", role_id_s, handler_s);
```

Comment:
I think the debugging message is inaccurate

Reply:
Just wondering how building process got passed.